extensions/wikia/ArticleVideo/styles/jwplayer.scss

```diff
@@ -0,0 +1,105 @@
+@import 'video-shared-variables';
+
+.featured-video {
+	padding-top: 56.25%; // we want to have 16:9 ratio div size
+	position: relative;
+	width: 100%;
+
+	&__player-container {
+		bottom: 0;
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+		transform: translate3d(0, 0, 0); // hack for iOS for changing position to fixed while scrolling
+		z-index: 2;
+	}
+
+	&.is-collapsed-ready &__player-container {
+		left: auto;
+		position: fixed;
+		top: auto;
+	}
+
+	&.is-collapsed &__player-container {
+		bottom: 45px !important;
+		left: auto;
+		position: fixed;
+		right: 20px !important;
+		top: auto;
+		transition: right $video-collapse-animation-speed, bottom $video-collapse-animation-speed, width $video-collapse-animation-speed;
+		width: $video-container-width !important;
+	}
+
+	&.is-collapsed &__details {
+		display: block;
+	}
+
+	&.is-collapsed &__close {
+		cursor: pointer;
+		display: block;
+		z-index: 11001;
+	}
+
+	&__details {
+		background-color: #000;
+		display: none;
+		font-weight: bold;
+		padding: $video-details-horizontal-padding $video-details-vertical-padding;
+		-moz-osx-font-smoothing: grayscale;
```

Comment:
Seems like these should go abo the font-weight declaration:
https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#propertysortorder